lib/typing/ext/variables.go

```diff
@@ -16,6 +16,7 @@ var supportedDateTimeLayouts = []string{
 	RFC3339Nanosecond,
 	time.RFC3339Nano,
```

Comment:
Just noticed that this one is in here twice

Reply:
Nice!

Reply:
I'm gonna write a test to make sure it's only used once

Reply:
:+1: